index.js

```diff
@@ -6388,11 +6389,11 @@ export async function submitBom(args, bomContents) {
         console.log(
           "Unable to submit the SBOM to the Dependency-Track server using POST method",
         );
-        console.log(error);
       }
     } else {
       console.log("Unable to submit the SBOM to the Dependency-Track server");
-      console.log(error);
     }
+    console.log(error.response?.body);
```

Comment:
It appears like we are logging the message in two places - here as well as in bin/cdxgen.js.

Reply:
@prabhu Indeed. The difference is that the whole body is logged in index.js while only the error is logged in cdxgen.js.
You'd remove the log in cdxgen.js? Or remove from index.js and maybe add one in server.js?